.github/workflows/crossplane-release-refresh.yml

```diff
@@ -0,0 +1,68 @@
+name: Trigger PR for cross plane release
+
+#on:
+#  release:
+#    types: [published]  # Trigger when a new release is published
+on:
+  workflow_dispatch:
+
+jobs:
+  update-provider:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Set Up Git
+        run: |
+          git config --global user.name 'github-actions[bot]'
+          git config --global user.email 'github-actions[bot]@users.noreply.github.com'
+
+      - name: Clone provider-palette repository
+        run: git clone https://github.com/crossplane-contrib/provider-palette.git
+
+      - name: Get Latest Release
+        id: get_release
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const owner = 'spectrocloud';
+            const repo = 'terraform-provider-spectrocloud';
+            const latestRelease = await github.rest.repos.getLatestRelease({
+              owner,
+              repo,
+            });
+            core.setOutput('latest_version', latestRelease.data.tag_name);
+            console.log(`Latest release tag: ${latestRelease.data.tag_name}`);
+
+      - name: Update Makefile
+        run: |
+          cd provider-palette
+          sed -i 's/^export TERRAFORM_PROVIDER_VERSION := .*/export TERRAFORM_PROVIDER_VERSION := ${{ steps.get_release.outputs.latest_version }}/' Makefile
+
+      - name: Install goimports
+        run: go install golang.org/x/tools/cmd/goimports@latest
+
+      - name: Run make commands
+        run: |
+          cd provider-palette
+          make submodules vendor vendor.check
+          make generate
+
+      - name: Commit and Push Changes
+        run: |
+          cd provider-palette
+          git checkout -b update-terraform-provider-${{ github.event.release.tag_name }}
+          git add Makefile
+          git add .
+          git commit -m "Update Terraform Provider to version ${{ github.event.release.tag_name }}"
+          git push origin update-terraform-provider-${{ github.event.release.tag_name }}
```

Comment:
This is going to fail unless you configure auth using a PAT that has push access for provider-palette.